src/main/resources/languages/researches_en.yml

```diff
@@ -230,4 +230,4 @@ slimefun:
   magnesium_generator: Power from Magnesium
   kelp_cookie: Tasty Kelp
   makeshift_smeltery: Improvised Smeltery
-  tree_growth_accelerator: Faster Trees
+  tree_growth_accelerator: Faster Trees
```

Comment:
Aside from all of the above issues that are still waiting, this is also still missing an entry for your new research. There are no file changes.

Reply:
I thought i added that, pushing all changes now.
(my IDE has been acting up recently, either that or I'm just not good with version control)